src/components/SpecialResolution/Resolution.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <v-card
+    v-if="showSpecialResolutionResolution"
+    id="resolution"
+    flat
+  >
+    <!-- Header -->
+    <article class="header-container section-container">
+      <v-icon color="appDkBlue">
+        mdi-handshake
+      </v-icon>
+      <label class="font-weight-bold pl-2">Special Resolution</label>
+    </article>
+
+    <v-card
+      flat
+      :class="{'invalid-section': invalidResolutionSection}"
+    >
+      <InstructionalText />
+
+      <HelpResolution />
+
+      <section
+        id="resolution-section"
+        class="section-container"
+      >
+        <div
+          id="resolution-header"
+        >
+          <v-row
+            no-gutters
+            class="mt-4"
+          >
+            <v-col>
+              <h2>Special Resolution</h2>
+              <v-chip
+                v-if="isCoopCorrectionFiling && hasChanged"
+                id="resolution-corrected-lbl"
+                x-small
+                label
+                color="primary"
+                text-color="white"
+              >
+                {{ getEditedLabel }}
+              </v-chip>
+            </v-col>
+            <v-col class="align-right mt-n2">
+              <div
+                class="actions mr-4"
+              >
+                <v-btn
+                  v-if="!isEditing && hasChanged && isCoopCorrectionFiling"
+                  id="btn-resolution-undo"
+                  text
+                  color="primary"
+                  class="undo-action"
+                  @click="undoSpecialResolutionStore(); isEditing = false"
+                >
+                  <v-icon small>
+                    mdi-undo
+                  </v-icon>
+                  <span>Undo</span>
+                </v-btn>
+                <v-btn
+                  v-if="!isEditing && !hasChanged && isCoopCorrectionFiling"
+                  id="btn-change-resolution"
+                  text
+                  color="primary"
+                  @click="isEditing = true"
+                >
+                  <v-icon small>
+                    mdi-pencil
+                  </v-icon>
+                  <span>{{ getEditLabel }}</span>
+                </v-btn>
+              </div>
+            </v-col>
+          </v-row>
+        </div>
+        <p
+          v-if="isEditing"
+          class="section-description mt-2 info-text"
+        >
+          Enter the date the special resolution passed and the text as it appears on your printed form.
+        </p>
+        <v-form ref="resolutionForm">
+          <ResolutionEditor
+            ref="resolutionEditor"
+            :isEditing="isEditing"
+          />
+          <SigningParty
+            ref="signingParty"
+            :isEditing="isEditing"
+          />
+          <v-row
+            v-if="isEditing"
+            id="resolution-confirmation-buttons"
+            no-gutters
+            class="justify-end pr-8 pb-8 mt-8"
+          >
```

Comment:
What's the v-row used for, here, since there are no columns? Would a div with `flex` suffice?